app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java

```diff
@@ -81,30 +82,39 @@ protected void setupWithStatus(StatusViewData.Concrete status, final StatusActio
 
                 String rebloggedByDisplayName = status.getRebloggedByUsername();
                 if (rebloggedByDisplayName == null) {
-                    hideRebloggedByDisplayName();
+                    hideStatusInfo();
                 } else {
                     setRebloggedByDisplayName(rebloggedByDisplayName);
+                    statusInfo.setOnClickListener(v -> listener.onOpenReblog(getAdapterPosition()));
```

Comment:
Listener is set here but not replaced for poll?

Reply:
Agreed, its ugly, but its save because its are never used on the same ViewHolder. Will refactor once I have more time.